web/app/themes/gds/resources/blocks/tabs/tabs.blade.php

```diff
@@ -0,0 +1,24 @@
+@if( $innerBlocks ?? false )
+  <div {!! get_block_wrapper_attributes() !!}>
+    <div class="wp-block-buttons" role="tablist">
+      @foreach( $innerBlocks as $key => $tab )
+        <div class="wp-block-button" role="tab">
+          <button class="wp-block-button__link {{ $key === 0 ? 'active' : '' }}"
+                  type="button"
+                  role="presentation"
```

Comment:
```suggestion
                  role="tab"
```

The way you have it set up now the div above is the tab, but the listeners are on the button inside so if you click the `tab` nothing will happen.